src/promql.ts

```diff
@@ -60,6 +61,7 @@ export const promql = {
   unless: ({ left, right }: LogicalOpParams) => `${left} unless ${right}`,
 
   rate: ({ expr, interval = '$__rate_interval' }: Rate) => `rate(${expr}[${interval}])`,
+  increase: ({ expr, interval = '$__range' }: Increase) => `increase(${expr}[${interval}])`,
```

Comment:
Please create a unit test, see `src/test/rate.spec.ts` for an example ❤️

Reply:
Done!